src/opnsense/mvc/app/views/layouts/default.volt

```diff
@@ -306,37 +306,37 @@
     </div>
 
     <script>
-    /* hook translations  when all JS modules are loaded*/
```

Comment:
how are these translations handled in tabulator?

Reply:
See https://github.com/opnsense/core/pull/8555/commits/897fba1d84c1959d3712814b95eda2fb85cfc636